server/src/main/java/com/microsoft/java/bs/core/internal/managers/BuildTargetManager.java

```diff
@@ -68,8 +68,8 @@ public List<BuildTargetIdentifier> store(GradleSourceSets gradleSourceSets) {
             false /* canTest */,
             false /* canRun */,
             false /* canDebug */
-          )
-      );
+        )
+    );
```

Comment:
In this PR, there are some changes due to the format change like here. I suggest to revert changes like this to limit the size of the PR.

// Sorry that the repo does not have a format profile right now, we can add some later as an engineering work.

Reply:
Hi @jdneo , I tried to take care of it in the previous commit but the accidental Shift+Alt+F presses brought them back. Reverted them back again.